dipy/testing/__init__.py

```diff
@@ -11,14 +11,30 @@
 IO_DATA_PATH = abspath(pjoin(dirname(__file__),
                              '..', 'io', 'tests', 'data'))
 
-# Allow failed import of nose if not now running tests
-try:
-    import nose.tools as nt
-except ImportError:
-    pass
-else:
-    from nose.tools import (assert_equal, assert_not_equal,
-                            assert_true, assert_false, assert_raises)
+
+def assert_operator(value1, value2, msg="", op=operator.eq):
+    """Check Boolean statement."""
+    try:
+        if op == operator.is_:
+                value1 = bool(value1)
+        assert op(value1, value2)
+    except AssertionError:
+        raise AssertionError(msg.format(str(value2), str(value1)))
+
+
+assert_greater_equal = partial(assert_operator, op=operator.ge,
```

Comment:
Are these defined here because pytest doesn't have these? Looks like something that might be useful in numpy?

Reply:
`Numpy` and `Pytest` do not have them. they use `assert x > y`, but the error message is never explicit, it returns `True or False`. And yes, it can be useful for them.